exercises/leap/test/leap_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
     });
 
     test("is introduced every 4 years to adjust about a day", () {
-      final year = new Year(2020);
+      final year = new Year(1996);
```

Comment:
This change was made in a separate PR that was already merged into master, no need to include it here.